docs/source/usage/cli.rst

```diff
@@ -78,6 +78,16 @@ Users can view available offers and claim an offer to create a lease.
 | Delete Lease | ``openstack esi lease delete <lease_uuid>``                                                |
 +--------------+--------------------------------------------------------------------------------------------+
 
+**Admin Actions**
+
+Admins can update the end time of a lease.
+
++--------------+-----------------------------------------------------------------------+
+|              | **Admin Actions**                                                     |
++--------------+-----------------------------------------------------------------------+
+| Update Lease | ``openstack esi lease update --end-time <new_end_time> <lease_uuid>`` |
```

Comment:
Can you clarify that `new_end_time` > `current_end_time`?

Reply:
Oh, this may have been unclear before; new_end_time can be less than current_end_time. When that happens, esi_leap does not do any time verification because we *know* the new time span will be acceptable since it's within the existing lease.